### ISSUE TITLE
Cursor Focus - Main Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OSHA-Heat-Safety-iOS#2]
Once in edit mode in either the temperature or humidity fields the focus should be on the "Set" button or the number keyboard to make entering information easier for those using VoiceOver.

WCAG 2.0 -3.2.1 - On Focus - Level A

**Comment by MikePulsiferDOL** [comment]
Please compare to the behavior in stock apps.  These aren't web pages.
